docs/openshift-installation.md

```diff
@@ -0,0 +1,18 @@
+# Openshift Installation
```

Comment:
just to be more specific. This is how this doc is referenced in the docs/README.md
```suggestion
# Installation in an Openshift cluster the OLM.
```